deploy/static/with-rbac.yaml

```diff
@@ -32,6 +32,7 @@ spec:
             - --udp-services-configmap=$(POD_NAMESPACE)/udp-services
             - --publish-service=$(POD_NAMESPACE)/ingress-nginx
             - --annotations-prefix=nginx.ingress.kubernetes.io
+            - --v=2
```

Comment:
This is the manifest users use to deploy by default right? Why are you increasing verbosity?

Reply:
I will remove that before merging the PR